controls/roles/validator-import-api/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 - block:
   - name: Get validator service's auth-token
     slurp:
-      src: "/opt/app/services/{{ validator_service }}/data/wallets/auth-token"
+      src: "{{ auth_token_path }}"
```

Comment:
passing full path instead of validator id because of variable service paths.
i think there are no resources that i could extract that information from

Reply:
Either Ansible pulls the information from the volumes or JS does it. You can look up the volumes, look for where the volume is set to and then build the correct path with the information.

Reply:
I didn't want to make huge changes but i'd prefer doing it in ansible

Reply:
I'll have a look at it!

Reply:
see https://github.com/NeoPlays/ethereum-node/pull/1

Reply:
Ahhh perfect, thanks a lot 🙌🏼